cognite/client/_api/entity_matching.py

```diff
@@ -0,0 +1,220 @@
+        return self._retrieve_multiple(ids=ids, external_ids=external_ids, wrap_ids=True)
+
+    def update(
+        self,
+        item: Union[
+            EntityMatchingModel, EntityMatchingModelUpdate, List[Union[EntityMatchingModel, EntityMatchingModelUpdate]]
+        ],
+    ) -> Union[EntityMatchingModel, List[EntityMatchingModel]]:
+        """Update model
+
+        Args:
+            item (Union[EntityMatchingModel,EntityMatchingModelUpdate,List[Union[EntityMatchingModel,EntityMatchingModelUpdate]]) : Model(s) to update
+        """
+        return self._update_multiple(items=item)
+
+    def list(self, filter: Dict = None, limit=100) -> EntityMatchingModelList:
+        """List models
+
+        Args:
+            filter (dict): If not None, return models with parameter values that matches what is specified in the filter.
+            limit (int, optional): Maximum number of items to return. Defaults to 100. Set to -1, float("inf") or None to return all items.
+
+        Returns:
+            EntityMatchingModelList: List of models."""
+        filter = {utils._auxiliary.to_camel_case(k): v for k, v in (filter or {}).items() if v is not None}
+        # NB no pagination support yet
+        models = self._post(self._RESOURCE_PATH + "/list", json={"filter": filter}).json()["items"]
+        return EntityMatchingModelList(
+            [self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client) for model in models]
+        )
+
+    def list_jobs(self) -> EntityMatchingModelList:
+        """List jobs
+
+        Returns:
+            EntityMatchingModelList: List of jobs."""
+        return EntityMatchingModelList(
+            [
+                self._LIST_CLASS._RESOURCE._load(model, cognite_client=self._cognite_client)
+                for model in self._get(self._RESOURCE_PATH + "/jobs").json()["items"]
+            ]
+        )
+
+    def delete(self, id: Union[int, List[int]] = None, external_id: Union[str, List[str]] = None) -> None:
+        """Delete models
+
+        Args:
+            id (Union[int, List[int]): Id or list of ids
+            external_id (Union[str, List[str]]): External ID or list of external ids"""
+        self._delete_multiple(ids=id, external_ids=external_id, wrap_ids=True)
+
+    def fit(
+        self,
+        sources: List[Union[Dict, CogniteResource]],
+        targets: List[Union[Dict, CogniteResource]],
+        true_matches: List[Union[Dict, Tuple[Union[int, str], Union[int, str]]]] = None,
+        match_fields: List[Tuple[str, str]] = None,
+        feature_type: str = None,
+        classifier: str = None,
+        ignore_missing_fields: bool = False,
+        name: str = None,
+        description: str = None,
+        external_id: str = None,
+    ) -> EntityMatchingModel:
+        """Fit entity matching model.
+        Note: All users on this CDF subscription with assets read-all capability in the project, are able to access the data sent to this endpoint.
```

Comment:
I suppose this note should be updated once the em acls are in place? which should happen before going to beta.

Reply:
have changed it to entitymatching read and ...